lib/WorldState.ts

```diff
@@ -266,8 +265,9 @@ export class WorldState {
 
   /**
    * Midrath cycle (soulframe)
+   * Game seems to desync while you play and appearntly it changes between updates.
    */
-  midrathCycle: MidrathCycle;
+  // midrathCycle: MidrathCycle;
```

Comment:
why's this removed now?

Reply:
Turns out the in-game timer is actually pretty buggy, joined in with a friend and realezied my timer wasn't the same as thiers so there's a chance that for the time being the timer itself might be different between players. That and found out that DE is also still tweaking the time, so figured keep the code and let them bake the game a bit more